lang/en.lyaml

```diff
@@ -717,6 +714,12 @@ en:
           examples:
             default: "Create a component within your project"
             withFlags: "Use --name and --type flags to bypass the prompt."
+        migrate:
+            describe: "Migrate a project to the projects framework."
```

Comment:
Does this copy need to be updated? Assuming this came from "migrate an app to the projects framework" but now it doesn't make a ton of sense